src/chunk.c

```diff
@@ -1264,6 +1264,11 @@ chunk_create_from_hypercube_and_table_after_lock(const Hypertable *ht, Hypercube
 		CommandCounterIncrement();
 	}
 
+	// TODO Well!  OSM's move_chunk_to_timescaledb DOES need the check constraints added here (we
```

Comment:
we should add the check constraints in chunk_api.c .

Reply:
Done.  It's almost entirely unchunkinized, and what remains is not exposed, so we can finish unchunkinizing later without disturbing callers.  I didn't want to try that without discussing with someone with some ownership here, as there's more than one way to go.